theano/sandbox/cuda/dnn.py

```diff
@@ -1731,6 +1746,8 @@ def c_code(self, node, name, inputs, outputs, sub):
                cudnnGetErrorString(err));
   %(fail)s
 }
+
+}
```

Comment:
As you didn't changed the indentation, add a comment to tell which if it is matching to help reader of the code.